lib/rules/require-ember-lifeline.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileOverview Require the use of ember-lifeline's runTask over using Ember.run.later.
+ * @author Steve Calvert
+ */
+
+const { getCaller, cleanCaller } = require('../utils/caller');
+const { getEmberImportBinding } = require('../utils/imports');
+const { collectObjectPatternBindings } = require('../utils/destructed-binding');
+
+let LATER_PATTERNS = ['Ember.run.later', 'run.later'];
+
+const MESSAGE = 'Please use runTask from ember-lifeline in favor of Ember.run.later.';
+
+module.exports = {
+  docs: {
+    description: 'Please use runTask from ember-lifeline in favor of Ember.run.later.',
```

Comment:
As you can see now, the message is dynamically generated based on the node being inspected. I've adjusted it now.